components/Header.tsx

```diff
@@ -110,197 +105,182 @@ const Header: React.FC<Props> = ({
                 zIndex: 200,
                 position: 'sticky',
                 top: 0,
-                padding: '12px 40px',
+                padding: '15px 40px',
                 backgroundColor: 'var(--docs_bg_header)',
                 borderBottom: '1px solid var(--docs_border_default)',
                 boxSizing: 'border-box',
                 gap: '40px',
                 '@md': {
-                    padding: '20px 24px'
+                    padding: '16px 24px'
                 }
             }}>
-            <div className="head-left">
-                <a href="https://www.100ms.live" style={{ display: 'flex', marginRight: '40px' }}>
+            <Flex align="center">
+                <a
+                    target="_blank"
+                    rel="noreferrer"
+                    onClick={() =>
+                        window.analytics.track('link.clicked', {
+                            btnId: 'logo.clicked',
+                            currentPage: window.location.href
```

Comment:
window.location.pathname?

Reply:
More data is better? Can refine further on amplitude to only get prod events